### ISSUE TITLE
GitLens premium walkthrough images missing

### ISSUE THREAD

**Issue by misolori** [gitkraken/vscode-gitlens#1838]
![image](https://user-images.githubusercontent.com/35271042/155766308-007c44c3-8fd7-49cc-8ff2-b4c47c38669f.png)

👍  to using the walkthroughs, but noticed that the images in the walkthrough all appear missing/broken

- GitLens Version: v2022.2.2417 (Insiders)
- Git Version: 2.32.0 (Apple Git-132)

Version: 1.65.0-insider (Universal)
Commit: 8440b8f47231147823ba9696913d642e5f2d9d58
Date: 2022-02-25T05:18:40.990Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin arm64 21.3.0

**Comment by misolori** [comment]
Actually this might be a bug on VS Code as images are not showing up but svgs/gifs are...

**Comment by eamodio** [comment]
@misolori Yeah, seems to only be happening in Insiders, stable VS Code seems fine. Do you know if there is an open issue?

**Comment by misolori** [comment]
Yes, we'll be pulling this as a candidate since we are shipping this week, being tracked here https://github.com/microsoft/vscode/issues/143985

**Comment by eamodio** [comment]
@misolori awesome thank you!
